docs/commands/data.json

```diff
@@ -640,9 +640,19 @@
         "arg": "--path-prefix-major-version <path-prefix-major-version>",
         "description": "Version to be used in the path prefix; will be used to configure Traefik2 IngressRoutes. ie. 'v1' will result in a path prefix of '/v1/servicename",
         "defaultValue": ""
+      },
+      {
+        "arg": "--service-build-vg <comma-delimitated-list-of-variable-group-names>",
+        "description": "List of variable groups to be linked to service pipeline. i.e. bedrock-vg,fabrikam-vg",
```

Comment:
These `description` for these don't match whats in `src/commands/service/create.decorator.json`. should they?

Reply:
they should match now!